drivers/usb/device/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ zephyr_sources_ifdef(CONFIG_USB_DW       usb_dc_dw.c)
 zephyr_sources_ifdef(CONFIG_USB_DC_STM32 usb_dc_stm32.c)
 zephyr_sources_ifdef(CONFIG_USB_DC_SAM0  usb_dc_sam0.c)
 zephyr_sources_ifdef(CONFIG_USB_DC_SAM   usb_dc_sam.c)
-zephyr_sources_ifdef(CONFIG_USB_NRF52840 usb_dc_nrfx.c)
+zephyr_sources_ifdef(CONFIG_USB_NRF52    usb_dc_nrfx.c)
```

Comment:
@barsok usb_dc_nrfx.c is common for nRF52 and nRF53. So you need a Kconfig switch that makes sense for both families.

Reply:
@carlescufi @SebastianBoe @anangl @emob-nordic what's your take here? How should the Kconfig option be called, so makes sense for nRF52 and nRF53 series? Note that the same driver will be used in both SoC families.

Reply:
I replied in a separate comment.

Reply:
If we have same IP core in both nRF52 and nRF53 families I would opt for USB_NRFX to be consistent with other Kconfig options related to drivers.

Reply:
CONFIG_USB_NRF ?